src/layer/tile/TileLayer.js

```diff
@@ -114,7 +114,13 @@ export var TileLayer = GridLayer.extend({
 
 	// @method setUrl(url: String, noRedraw?: Boolean): this
 	// Updates the layer's URL template and redraws it (unless `noRedraw` is set to `true`).
+	// If the URL does not change, the layer's URL template will not be redrawn unless

```

Comment:
I would phrase this as "[...] the layer will not be redraw unless [...]", since it's in fact the layer, not the URL template that is redrawn.